pkg/mimedb/db.go

```diff
@@ -1508,10 +1588,18 @@ var DB = map[string]struct {
 		ContentType:  "image/jpeg",
 		Compressible: false,
 	},
+	"jpf": {
```

Comment:
Is this supposed to be `jpx`?

Reply:
These are auto-generated @kannappanr and also jpf is `JPEG 2000 Image`